examples/cli.ini

```diff
@@ -0,0 +1,22 @@
+# This is an example of the kind of things you can do in a configuration file
+
+# Use a 4096 bit RSA key instead of 2048
+rsa-key-size = 4096
+
+# Always use the staging/testing server
+server = https://acme-staging.api.letsencrypt.org/directory
```

Comment:
FYI this will not stop people from overriding using `--server` on the CLI, if that's what you had in mind.